container/options.go

```diff
@@ -403,3 +409,33 @@ func Bottom(opts ...Option) BottomOption {
 		return opts
 	})
 }
+
+type paddingOption func() []Option
```

Comment:
Do we need this type for anything? Looks like we should be good with the already existing definition of Option.

Reply:
Oops, thats useless, I was testing some other thing. Will remove.